lib/relation-definition.js

```diff
@@ -2081,9 +2081,50 @@ EmbedsOne.prototype.related = function(condOrRefresh, options, cb) {
   }
 };
 
+EmbedsOne.prototype.prepareEmbeddedInstance = function(inst) {
+  if (inst && inst.triggerParent !== 'function') {
+    var self = this;
+    var propertyName = this.definition.keyFrom;
+    var modelInstance = this.modelInstance;
+    if (this.definition.options.persistent) {
+      var pk = this.definition.keyTo;
+      inst.__persisted = !!inst[pk];
+    } else {
+      inst.__persisted = true;
+    }
+    inst.triggerParent = function(actionName, callback) {
+      if (actionName === 'save' || actionName === 'destroy') {
+        var embeddedValue = self.embeddedValue();
+        modelInstance.updateAttribute(propertyName,
+          embeddedValue, function(err, modelInst) {
```

Comment:
When "destroy" was called, I think we need to unset the embedded value (or at least set it to `null`).

Please add a unit-test to verify this scenario.